pkg/controller/node/nodecontroller.go

```diff
@@ -193,6 +193,8 @@ func NewNodeController(
 			AddFunc:    nc.maybeDeleteTerminatingPod,
 			UpdateFunc: func(_, obj interface{}) { nc.maybeDeleteTerminatingPod(obj) },
 		},
+		// We don't need to build a index for podStore here
+		cache.Indexers{},
```

Comment:
For consistency, can you add `cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc}`.  It will ensure that if people start making use of the `podStore` in more specific ways, they'll get the benefits they expect.  It will also reserve the name for future refactorings.